src/Command/PelagosMigrateDatasetFilesCommand.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Dataset;
+use App\Entity\DatasetSubmission;
+use App\Entity\File;
+use App\Entity\FileSet;
+use App\Message\DatasetSubmissionFiler;
+
+use Doctrine\ORM\EntityManagerInterface;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+class PelagosMigrateDatasetFilesCommand extends Command
+{
+    protected static $defaultName = 'pelagos:migrate-dataset-files';
+
+    /**
+     * A Doctrine ORM EntityManager instance.
+     *
+     * @var EntityManagerInterface $entityManager
+     */
+    protected $entityManager;
+
+    /**
+     * The message bus for dispatching the filer message.
+     *
+     * @var MessageBusInterface $messageBus
+     */
+    protected $messageBus;
+
+    /**
+     * This contains a list of submission IDs that need to be queued to filer.
+     *
+     * @var Array $submissionsListForFiler
```

Comment:
Need to change it to "array"

Reply:
I was confused with the PSR-12, but the guidance is to use lower case short (if possible, like int, bool) statements. I thought it was the other way around.